### DISCUSSION TITLE
How to increase transfer speed??

### DISCUSSION THREAD

**Discussion by dronacharya27** [localsend/localsend#1551] [Discussion (English, International)]
I am using localsend to transfer files from my android to mac, but I am getting very slow transfer speed like 2 to 3 MB/s. Is there any setting that I need to do in order to get high transfer speeds?

**Comment by Tienisto** [comment]
You can disable encryption on all of your devices (advanced setting)

**Reply by DavidWoli** [reply]
Why is it necessary to disable encryption for higher speeds? Is it not technically possible to have encryption and high speed at the same time?

**Comment by EnverOsmanov** [comment]
What network do you use?

I was using WiFi 2.4 GHz got speed ~10 MB/s. Switched to WiFi 5 GHz (same router, same devices: PC & laptop) got speed 50-60 MB/s.
